### PR TITLE
ENGDOCS-1733

### PR DESCRIPTION
**Pull request by aevesdocker**
q3 freshness 3/3

### CODE REVIEW

**Review by stephaurelio — APPROVED** [review]
LGTM ✨

File: content/desktop/install/windows-install.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ _For checksums, see [Release notes](../release-notes.md)_
 
   - 64-bit processor with [Second Level Address Translation (SLAT)](https://en.wikipedia.org/wiki/Second_Level_Address_Translation)
   - 4GB system RAM
-  - BIOS-level hardware virtualization support must be turned on in the
+  - Turn on BIOS-level hardware virtualization support in the
     BIOS settings. For more information, see
```
Comment (dvdksn) [inline content/desktop/install/windows-install.md:49]:
```suggestion
  - Enable hardware virtualization in BIOS. For more information, see
```

File: content/desktop/hardened-desktop/settings-management/_index.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ It is available with [Docker Desktop 4.13.0 and later](../../release-notes.md).
 
 ### Who is it for?
 
-- For Organizations who wish to configure Docker Desktop to be within their organization's centralized control.
-- For Organizations who want to create a standardized Docker Desktop environment at scale.
+- For organizations who wish to configure Docker Desktop to be within their organization's centralized control.
+- For organizations who want to create a standardized Docker Desktop environment at scale.
```
Comment (dvdksn) [inline content/desktop/hardened-desktop/settings-management/_index.md:21]: WDYT?

```suggestion
- For organizations that want to configure Docker Desktop to be within their organization's centralized control.
- For organizations that want to create a standardized Docker Desktop environment at scale.
```

**Review by dvdksn — COMMENTED** [review]
some ideas

**Review by dvdksn — APPROVED** [review]

**Pull request merged**